modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully upgrade the cluster even when upgrade release image validation fails and the cluster is reporting errors.
+
+[IMPORTANT]
+====
+Setting a CVO override (such as using `--force`) puts the entire cluster in an unsupported state. Reported issues must be reproduced after removing any overrides for support to proceed.
+====
```

Comment:
> Setting a CVO override (such as using `--force`) puts the entire cluster in an unsupported state. Reported issues must be reproduced after removing any overrides for support to proceed.

This is not correct from technical and support point view.

Here is the suggested text: 
`Use of --force is not recommended for updates. Please read the whole section to understand the risks of using force for updates.`